src/otk/context.py

```diff
@@ -83,10 +83,10 @@ def variable(self, name: str) -> Any:
 
         value = self._variables
 
-        for part in parts:
+        for i, part in enumerate(parts):
             if isinstance(value, dict):
                 if part not in value:
-                    raise TransformVariableLookupError(f"Could not find {parts!r}")
+                    raise TransformVariableLookupError(f"could not find {parts!r}")
```

Comment:
(super nitpick) this is just done for consistency - either all errors should be capitalized or none, I opted for lowecase but I have no strong opinion as long as we are consistent